### ISSUE TITLE
Problem with SpectrumObservationStacker class

### ISSUE THREAD

**Issue by AtreyeeS** [gammapy/gammapy#1282]
There seems to some issue with the SpectrumObservationStacker class.

Stacked observations have a significantly steeper index and a lower amplitude.
for example:
For a dummy background and instrument response, I combine 2 similar spectra, each with fitted values:

```
Fit result info 
--------------- 
Model: PowerLaw

Parameters: 

	   name     value     error         unit      min max frozen
	--------- --------- --------- --------------- --- --- ------
	    index 2.105e+00 1.466e-01                 nan nan  False
	amplitude 3.743e-12 5.631e-13 1 / (cm2 s TeV) nan nan  False
	reference 1.000e+00 0.000e+00             TeV nan nan   True

Covariance: 

	name/name   index   amplitude
	--------- --------- ---------
	    index    0.0215 -1.46e-14
	amplitude -1.46e-14  3.17e-25 

Statistic: 55.222 (wstat)
Fit Range: [  1.00000000e-02   1.00000000e+02] TeV
```

After stacking, the fitted parameters of the stacked fit are:

```
Fit result info 
--------------- 
Model: PowerLaw

Parameters: 

	   name     value     error         unit      min max frozen
	--------- --------- --------- --------------- --- --- ------
	    index 2.298e+00 1.667e-01                 nan nan  False
	amplitude 2.942e-12 4.895e-13 1 / (cm2 s TeV) nan nan  False
	reference 1.000e+00 0.000e+00             TeV nan nan   True

Covariance: 

	name/name   index   amplitude
	--------- --------- ---------
	    index    0.0278 -5.34e-14
	amplitude -5.34e-14   2.4e-25 

Statistic: 93.805 (wstat)
Fit Range: [  1.00000000e-02   1.00000000e+02] TeV
```

This is happening systematically for all simulations.

**Comment by AnneLemiere** [comment]
I found a problem in the writting of alpha tables and backscale tables in "stack_backscal(self):"
I corrected it in an updated version of spectrum/observation.py can be found in gammapy/AnneLemiere/      in my "first_branch".
After some test with fake IRFS and simulations, it seems to fix the problem. 
Please dont hesitate to make some more tests with real data.

**Comment by AtreyeeS** [comment]
I tested Anne's changes and it now seems to give correct results!

**Comment by JouvinLea** [comment]
could you do a PR with these changes?

**Comment by AnneLemiere** [comment]
ok,I will do a PR this afternoon. 



De: "JouvinLea" <notifications@github.com> 
À: "gammapy/gammapy" <gammapy@noreply.github.com> 
Cc: "AnneLemiere" <alemiere@apc.in2p3.fr>, "Comment" <comment@noreply.github.com> 
Envoyé: Jeudi 8 Février 2018 10:58:05 
Objet: Re: [gammapy/gammapy] Problem with SpectrumObservationStacker class (#1282) 



could you do a PR with these changes?

**Comment by JouvinLea** [comment]
Si, let's just discuss this afternoon!

**Comment by registerrier** [comment]
Correction has been merged in PR #1319. Closing issue.
